prov/gni/include/gnix_ep.h

```diff
@@ -37,6 +37,9 @@
 #include "gnix.h"
 #include "gnix_nic.h"
 
+/* Default minimum multi receive buffer size. */
+#define GNIX_OPT_MIN_MULTI_RECV_DEFAULT	64
+
```

Comment:
This value is supposed to be able to be set/retrieved using fi_getopt/fi_setopt.
```
   fi_getopt / fi_setopt
       Endpoint protocol operations may be retrieved using fi_getopt or set using fi_setopt.  Applications specify the level that  a
       desired option exists, identify the option, and provide input/output buffers to get or set the option.  fi_setopt provides an
       application a way to adjust low-level protocol and implementation specific details of an endpoint.

       The following option levels and option names and parameters are defined.

       FI_OPT_ENDPOINT

       · FI_OPT_MIN_MULTI_RECV - size_t : Defines the minimum receive buffer space available when the receive  buffer  is  automati-
         cally  freed  (see  FI_MULTI_RECV).   Modifying  this  value  is  only guaranteed to set the minimum buffer space needed on
         receives posted after the value has been changed.  It is recommended that applications that want to  override  the  default
         MIN_MULTI_RECV value set this option before enabling the corresponding endpoint.
```